test/powershell/engine/ResourceValidation/TestRunner.ps1

```diff
@@ -36,6 +36,16 @@ function Test-ResourceStrings
     # This is the reason why this is not a general module for use. There is
     # no other way to run these tests
     Describe "Resources strings in $AssemblyName (was -ResGen used with Start-PSBuild)" -tag Feature {
+
+        function NormalizeLineEnd
```

Comment:
Maybe we should move this function here: https://github.com/PowerShell/PowerShell/blob/53e6ec6eadc2f363fc671e96b34ade4dda3fe8eb/test/tools/Modules/HelpersCommon/HelpersCommon.psm1

Reply:
There are different variations throughout the tests, so I thought keeping it local to the file is best.